### ISSUE TITLE
[DevEx]: Refactor Ranked Test in Game Over

### ISSUE THREAD

**Issue by Haviles04** [cuttle-cards/cuttle#825]
### Improvement Summary

Refactor some of the ranked test in Game Over Spec to use helpers

### Detailed Description

The `gameOver.spec.js` test file has two helper functions, `validateMatchResult` and `validateGameResult`, 
We should refactor the test starting on line 753 to utilize these helpers, and DRY it up.

**Comment by Haviles04** [comment]
No longer relevant after #852
